SDC/Postgres/schema.sql

```diff
@@ -0,0 +1,28 @@
+DROP DATABASE IF EXISTS reviews;
+
+CREATE DATABASE reviews;
+
+\c reviews;
+
+CREATE TABLE bnbUsers (
+  id SERIAL PRIMARY KEY,
+  username VARCHAR(50) NOT NULL,
+  profile_pic_url TEXT
+);
+
+CREATE TABLE bnbReview (
+  id SERIAL PRIMARY KEY,
+  message TEXT NOT NULL,
+  time_made TEXT NOT NULL,
```

Comment:
Why did you choose ```TEXT``` in this table for data types instead of ```VARCHAR```. Both work, but ```TEXT``` allows an unlimited length string. I think it is a little cleaner to stay consistent.

Reply:
I wanted to limit the number of characters for a username, but keep text length open.